ci/build-mingw64.sh

```diff
@@ -310,8 +313,10 @@ if [ "$2" = pack ]; then
         libgcc_*.dll lib{ssp,stdc++,winpthread}-[0-9]*.dll # compiler runtime
         av*.dll sw*.dll lib{ass,freetype,fribidi,harfbuzz,iconv,placebo}-[0-9]*.dll
         lib{shaderc_shared,spirv-cross-c-shared,dav1d}.dll zlib1.dll
-        # note: vulkan-1.dll is not here since drivers provide it
     )
+    if [[ "$TARGET" != "i686-"* ]]; then
```

Comment:
For completeness would be good to fix also 32bit build.

Reply:
Would you report/fix it upstream. I know they don't support mingw, so that's probably why it doesn't work. But would be nice to have 32-bit vulkan code tested somehow on our CI, MSYS build was removed, so only this one is left.

Reply:
I think it was some stdcall naming issue. How is shinchiro builds working?

Reply:
IIRC the issues on i686 did not relate to the assembly code at all.